packages/target-nodejs-sdk/src/index.js

```diff
@@ -231,12 +241,23 @@ export default function bootstrap(fetchApi) {
         },
         logger: this.logger,
         useBeacon: true,
-        ...options
+        ...request
       };
 
-      return executeDelivery(targetOptions).then(
-        preserveLocationHint.bind(this)
-      );
+      return executeDelivery(
+        targetOptions,
+        undefined,
+        this.telemetryProvider
```

Comment:
Let's move telemetryProvider to be the 2nd param for executeDelivery so that in this case we can just send 2 args and not explicitly use `undefined` for decisioningEngine